plugins/arclite/archive.cpp

```diff
@@ -400,6 +401,7 @@ void ArcAPI::load_codecs(const std::wstring& path) {
   const auto& add_hashers = [this](ArcLib &arc_lib, size_t lib_index) {
     if (arc_lib.ComHashers) {
       UInt32 numHashers = arc_lib.ComHashers->GetNumHashers();
+      arc_hashers.reserve(numHashers);
```

Comment:
Same as above: `add_hashers` is calling from a `for` loop below, and then again from a `while` loop.
`reserve`ing here would effectively kill exponential growth,